### DISCUSSION TITLE
Can i make the magnifier icon clickable?

### DISCUSSION THREAD

**Discussion by Sajjad-HTLO** [TarekRaafat/autoComplete.js#296] [Q&A]
Is it possible that i make the magnifier icon as a clickable button?
Or remove the icon and add my custom button instead of it?

**Comment by fidrelity** [comment]
Hi @Sajjad-HTLO. I had a similar issue. The icon is simply a background-image. You can remove it with a CSS override:

```
.autoComplete_wrapper>input {
  background-image:  none !important;
}
```

Then you could add basically any element as a button, for example wrap the input around a form and add a submit button.

Hope that helps ✌️
